Scripts/GameLoader.gd

```diff
@@ -0,0 +1,175 @@
+extends Node
+
+signal save_data_was_loaded
+
+const SAVE_PATH = "res://save.json"
+
+const OFFLINE_LIMIT : bool = false
+const OFFLINE_LIMIT_TIME : int = 7200 
+
+const OFFINE_BONUS_GOLD_RATIO : float = 0.3 
+const OFFINE_BONUS_XP_RATIO : float = 0.2
+
+var level_manager
+var elf
+var publican
+
+func setup() -> void:
+	for child in get_parent().get_children():
+		print(child.get_name())
+	level_manager = get_parent().get_node("World").get_node("LevelManager")
+	elf = get_parent().get_node("World").find_node("Elf")
+	publican = get_parent().get_node("World").find_node("Publican")
```

Comment:
I recommend to move initializing those variables into lines: 13, 14, 15, as wrote in the previous comment